draft-dns-zone-digest.xml

```diff
@@ -876,6 +897,41 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
 
         </texttable>
       </section>
+
+      <section title="&RRNAME; Hash Algorithm">
+        <t>
+          This document asks IANA to create a new
+          "&RRNAME; Hash Algorithm" registry
+          with initial contents as follows:
```

Comment:
"... future assignments are to be made through Expert Review [BCP26]"?

Reply:
I hadn't considered that before.  A quick read of BCP26 leads me to believe we would need to add some text as guidance to the expert reviewers?  i.e. we can't just say future assignments are through expert review and leave it at that?

Reply:
... future assignments are to be made through Expert Review [BCP26].
When making assignments, the expert reviewer should pay particular attention to ensuring that the request clearly specifies the canonical format and ordering of the input to the hash.  
---- ?

Actually, the more I think about this, the more I think "Specification Required" is more correct - this gets expert review, but also needs a stable place people can find out more. For 99% of assignments I'm assuming that the "specification" will be: "Like RFCxxx, but just use SHA-512, k?!", but for more complex things, which might need different canonicalization or sorting, an actual doc would be needed -- and "Expert Review, unless it's complex, then Specification Required" seems more faff than it is worth. We *could* reserve 1-128 for "DNSSEC canonicalization, fixed length hash" and Expert Review, and 129 - 240 for "Other sorting / variable length hash" and require Specifications, but that feels annoying. 
Whatever you want to do is fine....

Also, please add: 
        <c>0</c>
        <c>RESERVED</c>
        <c>RESERVED</c>
        <c>N/A</c>
        <c>N/A</c>